src/login7-payload.js

```diff
@@ -256,6 +316,53 @@ module.exports = class Login7Payload {
     return buffer.data;
   }
 
+  CreateFedAuthExtData(buffer, fedAuthInfo) {
```

Comment:
What do you think about changing this to `createFedAuthExtData ` to match the method naming convention?